CMakeLists.txt

```diff
@@ -385,8 +385,9 @@ else()
     set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-g")
     set(CMAKE_CXX_FLAGS_DEBUG  "-O0 -g")
     # These can be uncommented once we have visibility fully in place
```

Comment:
Remove this comment too since it
```suggestion
```

Reply:
I suppose comment should still be here since visibility is still default. I've changed it only for ngraph library